main.py

```diff
@@ -823,85 +825,289 @@ def on_link_down(self, event):
+                elif evc.id in flow_modifications:
+                    if "swap_to_failover" in flow_modifications[evc.id]:
+                        evc.failover_path = evc.current_path
+                        evc.current_path = evc.old_path
+                    else:
+                        evc.failover_path = evc.old_path
+                    evc.old_path = Path([])
+                    del flow_modifications[evc.id]
+                    del event_contents[evc.id]
+                    undeploy.append(evc)
 
-        if failover_event_contents:
-            emit_event(self.controller, "failover_link_down",
-                       content=deepcopy(failover_event_contents))
-        send_flow_mods_event(self.controller, switch_flows, 'install')
+            swap_to_failover_flows = {}
+            swap_to_failover_event_contents = {}
 
-        for evc in evcs_normal:
-            emit_event(
-                self.controller,
-                "evc_affected_by_link_down",
-                content={"link": link} | map_evc_event_content(evc)
-            )
+            clear_old_path_flows = {}
+            clear_old_path_event_contents = {}
 
-        evcs_to_update = []
-        for evc in evcs_with_failover:
-            evcs_to_update.append(evc.as_dict())
-            log.info(
-                f"{evc} redeployed with failover due to link down {link.id}"
-            )
-        for evc in check_failover:
-            evcs_to_update.append(evc.as_dict())
+            clear_old_path_reservations = list[EVC]()
 
-        self.mongo_controller.update_evcs(evcs_to_update)
+            for modified_evc_id in flow_modifications:
+                evc = evc_dict[modified_evc_id]
+                if "swap_to_failover" in flow_modifications[modified_evc_id]:
+                    swap_to_failover_flows = merge_flow_dicts(
+                        swap_to_failover_flows,
+                        flow_modifications[modified_evc_id]["swap_to_failover"]
+                    )
+                    swap_to_failover_event_contents[modified_evc_id] =\
+                        event_contents[modified_evc_id]["swap_to_failover"]
+                if "clear_old_path" in flow_modifications[modified_evc_id]:
+                    clear_old_path_flows = merge_flow_dicts(
+                        clear_old_path_flows,
+                        flow_modifications[modified_evc_id]["clear_old_path"]
+                    )
+                    clear_old_path_event_contents[modified_evc_id] =\
+                        event_contents[modified_evc_id]["clear_old_path"]
+                    clear_old_path_reservations.append(evc)
+                evcs_to_update.append(evc)
+
+            if swap_to_failover_flows:
+                self.execute_swap_to_failover(
+                    swap_to_failover_event_contents,
+                    swap_to_failover_flows
+                )
 
-        emit_event(
-            self.controller,
-            "cleanup_evcs_old_path",
```

Comment:
7) remove code that no longer is used [`on_cleanup_evcs_old_path` handler](https://github.com/kytos-ng/mef_eline/blob/fix/cleanup_old_path/main.py#L1147) that's subscribed to `kytos/mef_eline.cleanup_evcs_old_path` (and remove it from the README.rst too)

Reply:
Removed